COPYRIGHT

```diff
@@ -1,4 +1,3 @@
-Copyright 2008-2014 LSST Corporation
-Copyright 2016, 2017, 2019 Association of Universities for Research in Astronomy
+Copyright 2008-2014, 2016, 2017, 2019 Association of Universities for Research in Astronomy, Inc. (AURA)
```

Comment:
This wording is different from the form required by the Developer Guide: https://developer.lsst.io/legal/copyright-overview.html#copyright-holders

Reply:
Yes. Gemini told me that we are doing it wrong ☹️

Reply:
Maybe Gemini are doing it wrong! :-D

But if we are wrong — can we either fix the dev guide on this ticket, or file another ticket to do so?

Reply:
Gemini invoked "their lawyer told them" so I will fix the dev guide.

Reply:
The above format is what Gemini has always used. I think we once had a draft notice that just said "Association of Universities for Research in Astronomy" (maybe AURA in parentheses too) and Andy Flach, our contracts manager and lawyer on staff, asked me to change it to AURA's legal name, which includes the "Inc.". By the same logic, you might also have LSST, Inc., but I'll let you figure that one out...